index.html

```diff
@@ -0,0 +1,349 @@
+      <a href="" class="desk-abt desk-button">About</a>
+      <a href="" class="desk-contact desk-button"> Contact</a>
+      <nav class="desk-nav desk-button">
+        <img src="assets/images/Union.png" alt="M">
+      </nav>
+    </ul>
+
+  </header>
+  <section class="first-page">
+    <div class="headline">
+      <h1 class="hey">Hey There I'm Solomon</h1>
+      <h2 class="softwaredev"> I am software developer</h2>
+      <p class="headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="socio-image">
+      <ul class="socila-link">
+        <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+              alt="Github"></a></li>
+        <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+              src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+        <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+              alt="tiwtter"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+      </ul>
+      <img src="./assets/images/Group.png" alt="">
+    </div>
+  </section>
+  <section class="desk-first-page">
+    <div class="desk-headline">
+      <h1 class="desk-hey">Hey There I'm Solomon</h1>
+      <h2 class="desk-softwaredev"> I am software developer</h2>
+      <p class="desk-headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <ul class="socila-link">
+      <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+            alt="Github"></a></li>
+      <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+            src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+      <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+    <img src="./assets/images/Gourp1-desktop.png" alt="Image in the first page">
+  </section>
+  <section class="cards">
+    <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
+    </div>
+    <img class="main-image" src="assets/images/Placeholder.png" alt="main image">
```

Comment:
Let's make this section look like the design😉 compare the spacings around the horizontal rule and make the necessary change😉

yours
<img width="937" alt="your lines" src="https://user-images.githubusercontent.com/64643995/220944701-c98b25bf-0fd8-4b86-8e33-7ea419ea886d.png">

design
<img width="497" alt="design lines" src="https://user-images.githubusercontent.com/64643995/220944792-05ef2efb-eb1a-4d5f-a014-047ce399359b.png">